lib/src/mockserver/server.ts

```diff
@@ -66,6 +72,10 @@ export function runMockServer(
   };
 }
 
+function stripDraft(endpointName: string): string {
```

Comment:
I don't think `endpointName` is the correct name for this argument given its usage?

Reply:
Good point. Maybe `url` or `path` is more appropriate.